components/dot_menu/dot_menu.jsx

```diff
@@ -205,7 +207,12 @@ export default class DotMenu extends Component {
             const height = rect.height;
             const windowHeight = window.innerHeight;
             if ((y + height) > (windowHeight - MENU_BOTTOM_MARGIN)) {
-                this.setState({openUp: true});
+                this.setState({
+                    openUp: true,
```

Comment:
Nit: we can possibly flatten this conditional out but only setting the condition of height directly in this `openUp` property.

Reply:
Would you mind clarifying what is meant by `flatten out` in this case?

Reply:
Something like:

```
 this.setState({
                openUp: (y + height) > (windowHeight - MENU_BOTTOM_MARGIN) ? true : false,
                width: rect.width,
            });
```

Reply:
Awesome, thank you!  :) updated.